### DISCUSSION TITLE
Reverse geocoding difference in photon and nominatim

### DISCUSSION THREAD

**Discussion by technofection-rakesh** [komoot/photon#779] [Q&A]
Hi,

I have just downloaded the latest photon build (0.4.4) and downloaded the country extract of India from graphhopper. Now when I run a reverse geocoding on photon for the following lat, lon, photon is unable to give me any address, whereas the same coordinates gives reverse geocoding response on nominatim..

lat=22.599113&lon=88.209946

on photon the response is

{
    "features": [],
    "type": "FeatureCollection"
}

where as on nominatim, there response is

[
    {
        "place_id": 246993858,
        "licence": "Data © OpenStreetMap contributors, ODbL 1.0. http://osm.org/copyright",
        "osm_type": "way",
        "osm_id": 1098257265,
        "lat": "22.6000913",
        "lon": "88.2078351",
        "category": "highway",
        "type": "unclassified",
        "place_rank": 26,
        "importance": 0.10000999999999993,
        "addresstype": "road",
        "name": "",
        "display_name": "Domjur, Howrah District, West Bengal, 711411, India",
        "boundingbox": [
            "22.6000913",
            "22.6011983",
            "88.2052884",
            "88.2078351"
        ]
    }
]

Why is so?

**Comment by mtmail** [comment]
I don't see many named roads in the area. Even Donjur is huge https://www.openstreetmap.org/relation/10371827

https://photon.komoot.io/reverse?lon=88.209946&lat=22.599113

https://nominatim.openstreetmap.org/ui/reverse.html?lat=22.599113&lon=88.209946

**Reply by lonvia** [reply]
Indeed. Photon does not import roads without names in the first place, so you cannot find them with reverse geocoding either. Also be aware that reverse geocoding is limited because Photon does not import full geometries but only centroids.
